R/harmonic.R

```diff
@@ -1,7 +1,7 @@
 #' Add sin and cos terms for harmonic analysis
 #'
-#' `step_harmonic` creates a *specification* of a recipe step that
-#'   will add sin and cos terms for harmonic analysis.
+#' `step_harmonic()` creates a *specification* of a recipe step that will add
+#' sin and cos terms for harmonic analysis.
```

Comment:
```suggestion
#' `sin()` and `cos()` terms for harmonic analysis.
```

Or `$math font$`? I'm not sure.